closure/goog/net/xhrio.js

```diff
@@ -530,8 +539,9 @@ goog.net.XhrIo.prototype.send = function(
 
   // Set up upload/download progress events, if progress events are supported.
   if (this.getProgressEventsEnabled() && 'onprogress' in this.xhr_) {
-    this.xhr_.onprogress =
-        goog.bind(function(e) { this.onProgressHandler_(e, true); }, this);
+    this.xhr_.onprogress = goog.bind(function(e) {
```

Comment:
I am using clang-format version 4.0.0 and also confirmed that I use `style=Google` flag while formatting. When I execute `clang-format -i -style=Google foo.js bar.js` for the first time before commit, it did reformat the complete file. I think, I should have used the `git diff***` variant of the command instead.